lib/features/manage_account/presentation/configuration/vacation/vacation_view.dart

```diff
@@ -430,25 +438,33 @@ class VacationView extends GetWidget<VacationController> {
   }
 
   Widget _buildMessageBodyWidget(BuildContext context, double opacity) {
-    return (TextFieldBuilder()
-      ..key(const Key('message_body_editor'))
-      ..cursorColor(Colors.black)
-      ..addController(controller.messageBodyEditorController)
-      ..textStyle(TextStyle(
-          fontWeight: FontWeight.normal,
-          color: Colors.black.withOpacity(opacity),
-          fontSize: 16))
-      ..textDecoration((VacationInputDecorationBuilder()
-          ..setContentPadding(const EdgeInsets.symmetric(
-              vertical: BuildUtils.isWeb ? 16 : 12,
-              horizontal: 12))
-          ..setHintStyle(TextStyle(
-              color: AppColor.colorHintInputCreateMailbox.withOpacity(opacity),
+    return Obx(() {
+      return (TextFieldBuilder()
```

Comment:
please note that, vacation also support `htmlBody`. Please do it the same with composer

Reply:
but at the moment, no problem with only support `textBody`

Reply:
we will support `htmlBody` in https://github.com/linagora/tmail-flutter/issues/858

Reply:
oke